### PR TITLE
Help command - Update usage and options for native commands.

### PR DESCRIPTION
**Pull request by Or-Gabay**

### CODE REVIEW

File: artifactory/cli.go [hunk 1]
```diff
@@ -885,17 +883,42 @@ func getDockerFlags() []cli.Flag {
 	flags = append(flags, getSkipLoginFlag())
 	return flags
 }
+func getcommondepracatedFlages() []cli.Flag {
```
Comment (eyalbe4) [inline artifactory/cli.go]: getcommondepracatedFlages() --> getDepracatedFlages()

File: artifactory/cli.go [hunk 2]
```diff
@@ -885,17 +883,42 @@ func getDockerFlags() []cli.Flag {
...
+	return []cli.Flag{
 
-//These flags are not valid for native npm command
-func getNpmLegacyFlages() []cli.Flag {
-	npmFlags := []cli.Flag{
 		cli.StringFlag{
-			Name:  "npm-args",
-			Usage: "[Optional] A list of npm arguments and options in the form of \"--arg1=value1 --arg2=value2\"` `",
+			Name:  "user",
+			Usage: "[depracated][Optional] Artifactory username.` `",
```
Comment (eyalbe4) [inline artifactory/cli.go]: [depracated] --> [Depracated]

File: artifactory/cli.go [hunk 3]
```diff
@@ -885,17 +883,42 @@ func getDockerFlags() []cli.Flag {
...
+		cli.StringFlag{
+			Name:  "access-token",
+			Usage: "[depracated][Optional] Artifactory access token.` `",
+		},
+		cli.BoolFlag{
+			Name:  "insecure-tls",
+			Usage: "[depracated][Default: false] Set to true to skip TLS certificates verification.` `",
+		},
+		cli.StringFlag{
+			Name:  "url",
```
Comment (eyalbe4) [inline artifactory/cli.go]: It probably makes more sense to have the url as the first flag in this function.

File: artifactory/cli.go [hunk 4]
```diff
@@ -885,17 +883,42 @@ func getDockerFlags() []cli.Flag {
...
+		cli.StringFlag{
+			Name:  "apikey",
+			Usage: "[depracated][Optional] Artifactory API key.` `",
+		},
+		cli.StringFlag{
+			Name:  "access-token",
+			Usage: "[depracated][Optional] Artifactory access token.` `",
+		},
+		cli.BoolFlag{
+			Name:  "insecure-tls",
```
Comment (eyalbe4) [inline artifactory/cli.go]: The insecure-tls flag is not deprecated. It is however supported only by the upload command. It is added to all commands right now due to a bug, so I guess we should just remove it from this function.

File: docs/artifactory/mvn/help.go [hunk 5]
```diff
@@ -2,13 +2,10 @@ package mvn
 
 const Description = "Run Maven build."
 
-var Usage = []string{`jfrog rt mvn "<goals and options>" <config file path> [command options]`, `jfrog rt mvn "<goals and options> -f path/to/pom.xml" <config file path> [command options]`}
+var Usage = []string{`jfrog rt mvn "<goals and options>" [command options]`, `jfrog rt mvn "<goals and options> -f path/to/pom.xml" [command options]`}
```
Comment (eyalbe4) [inline docs/artifactory/mvn/help.go]: The usage still shows that the maven arguments are sent as a single argiment:
"[goals and options] -f path/to/pom.xml" --> [goals and options]
and 
"[goals and options]" --> [goals and options]

The same goes for the gradle

File: docs/artifactory/npminstall/help.go [hunk 6]
```diff
@@ -2,7 +2,4 @@ package npminstall
 
 const Description = "Run npm install."
 
-var Usage = []string{`jfrog rt npmi [command options] <repository name>`}
-
-const Arguments string = `	repository name
-		The source npm repository. Can be a local, remote or virtual npm repository.`
+var Usage = []string{`jfrog rt npmi [command options]`}
```
Comment (eyalbe4) [inline docs/artifactory/npminstall/help.go]: Missing *npm args* (see the usage of the go and nuget commands).

File: docs/artifactory/gradleconfig/help.go [hunk 7]
```diff
@@ -1,8 +1,5 @@
 package gradleconfig
 
-const Description = "Generate Gradle build configuration."
+const Description = "Generate gradle build configuration."
 
-var Usage = []string{"jfrog gradle-config <config file path>"}
-
-const Arguments string = `	config file path
-		Gradle build configuration file path.`
+var Usage = []string{"jfrog rt gradle-config [command options]"}
```
Comment (eyalbe4) [inline docs/artifactory/gradleconfig/help.go]: [command options] should removed.
This comment applies for all the config commands.

File: artifactory/cli.go (hunk 1 above)
Comment (Or-Gabay) [reply artifactory/cli.go]: Fixed.

File: artifactory/cli.go (hunk 2 above)
Comment (Or-Gabay) [reply artifactory/cli.go]: Fixed.

File: artifactory/cli.go (hunk 3 above)
Comment (Or-Gabay) [reply artifactory/cli.go]: Ok.

File: artifactory/cli.go (hunk 4 above)
Comment (Or-Gabay) [reply artifactory/cli.go]: Removed.

File: docs/artifactory/mvn/help.go (hunk 5 above)
Comment (Or-Gabay) [reply docs/artifactory/mvn/help.go]: Fixed

File: docs/artifactory/npminstall/help.go (hunk 6 above)
Comment (Or-Gabay) [reply docs/artifactory/npminstall/help.go]: Fixed.

File: docs/artifactory/gradleconfig/help.go (hunk 7 above)
Comment (Or-Gabay) [reply docs/artifactory/gradleconfig/help.go]: Don't we have a `-global` option?

**Pull request merged**